public class Demo3 {
    public static void main(String[] args) {
        
        int i,j;
        int n=5;

        for(i=1;i<=n;i++){
            for(j=1;j<=i+1;j++){ // updated line
                System.out.print("*");
            }
            System.out.println();
        }
    }
}
